packages/advanced-logic/specs/payment-network-any-declarative-0.2.0.md

```diff
@@ -347,6 +355,88 @@ The 'addPaymentInstruction' event:
 
 ---
 
+#### addPayeeDelegate
```

Comment:
the 2 methods could be merged in 1 `addDelegate`, which would either add a payee or payer delegate depending on who did the action